env.go

```diff
@@ -91,9 +101,10 @@ func (e *Env) Run(steward Steward) error {
 	sort.Strings(variables.Unset)
 
 	vals := templateVals{
-		Command: e.Command,
```

Comment:
One of my gripes about golang (fmt specifically) is the noisy diffs. Is it really necessary to indent everything masking the real change in the process.